packages/manager/modules/pci/src/projects/project/training/jobs/submit/submit.html

```diff
@@ -145,76 +146,112 @@ <h2>
-                        >
-                            <oui-select
-                                id="container"
-                                name="container"
-                                data-model="container"
-                                data-items="$ctrl.filteredContainers"
-                                data-match="description"
-                                data-searchable
-                                data-required
-                            >
-                            </oui-select>
-                        </oui-field>
-                    </oui-inline-adder-field>
-                    <oui-inline-adder-field>
-                        <oui-field
-                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_mount_path' | translate  }}"
-                        >
-                            <input
-                                type="text"
-                                class="oui-input"
-                                placeholder="/workspace/demo"
-                                data-ng-init="mountPath = '/workspace/container_' + $ctrl.job.volumes.length"
-                                name="mountPath"
-                                data-valid-mount="$ctrl.job.volumes"
-                                data-ng-model="mountPath"
-                                data-ng-pattern="/^\/(\S)*/"
-                                data-ng-model-options="{ debounce: 500 }"
-                                required
-                            />
-                        </oui-field>
-                    </oui-inline-adder-field>
-                    <oui-inline-adder-field data-adaptive>
-                        <oui-field
-                            data-label="{{:: 'pci_projects_project_training_jobs_submit_data_permission' | translate  }}"
-                            size="m"
-                        >
-                            <label class="oui-select">
-                                <select
-                                    name="permission"
-                                    ng-model="permission"
-                                    class="oui-select__input"
-                                    required
-                                >
-                                    <option
-                                        ng-repeat="perm in $ctrl.volumesPermissions"
-                                        value="{{perm}}"
-                                    >{{::
-                                        'pci_projects_project_training_jobs_submit_data_permission_'
-                                    + perm | translate }}
-                                    </option
+                <oui-switch data-model="$ctrl.job.addVolume" name="addVolume">
+                </oui-switch>
+            </oui-field>
+            <div class="oui-field__control" data-ng-if="$ctrl.job.addVolume">
+                <div class="oui-inline-adder mb-2">
```

Comment:
**question:** Out of curiosity, following template looks like what `<oui-inline-adder>` already provide.
Is there any reason to not rely on this component?

Reply:
There is a validation issue with the `<oui-inline-adder>` component. If the user has not selected anything, the button to add that is always active and the customer might think its already added but it wont be. 
So custom component here helps in adding validation and checking if the input is added in the form

Reply:
The expected display is the same but not the behavior... So @varun257 replace the component by styled html.